sources/packages/web/src/components/aest/OfferingChangeRequest.vue

```diff
@@ -34,7 +34,7 @@ export default defineComponent({
   //TODO: This emit needs to be removed when the program and offering header component
   //TODO: is enhanced to load header values with it's own API call.
   emits: ["getHeaderDetails"],
-  setup(props: any, context: SetupContext) {
+  setup(props, context: SetupContext) {
```

Comment:
Like props, context doesn't need to be explicitly typed.

![image](https://user-images.githubusercontent.com/54600590/236071225-6ddeed6d-6649-494a-85c8-4946d93fa054.png)